docs/feature-report/README.md

```diff
@@ -0,0 +1,553 @@
+# Feature report 
+
+## Why do we need TCS 2?
+
+The Taxon Concept Schema (TCS) is the TDWG standard for exchanging taxonomic
+data. It is one of the '2005' group of standards, together with Access to
+Biological Collections Data (ABCD) and Structured Descriptive Data (SDD), that
+consists of an XML Schema. Unlike ABCD, TCS did not have a dedicated group of
+users or maintainers and has not been maintained since shortly after it was
+ratified. Any problems that people might have with TCS were never fixed, because
+there was no mechanism for making changes to the XML Schema.
+
+As TCS is an XML Schema, data delivered as TCS has to be XML, which was the
+predominant format for data exchange at the time. A few years after TCS was
+ratified, another format, Comma-delimited Values (CSV), became people's
+preferred format for shipping around large amounts of data, at least in the
+Biodiversity Data domain. Another issue people have with TCS is that it can only
+be used for complete datasets and not for individual Taxon Concepts or Taxon
+Names. 
```

Comment:
Hmm. Didn't realize this was the case. Not sure why it is the case. I've often used the RDF version of TCS for single TCs.